docs/source/RunningTestsRemotely.rst

```diff
@@ -62,8 +62,10 @@ Once the remote machine is properly setup, you may run your test. Example::
     RESULTS    : PASS 1 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0
     TESTS TIME : 1.01 s
 
-As you can see, Avocado will copy the tests you have to your remote machine and
-execute them. A bit of extra logging information is added to your job summary,
+Notice that Avocado will not copy the tests to the remote machine. If you want
```

Comment:
I'll add either a section, or some more in-place information, about the caveats of Avocado's remote test copying support.